### ISSUE TITLE
Schemas added to namespace.models are dropped

### ISSUE THREAD

**Issue by erinpentecost** [noirbizarre/flask-restplus#685]
When adding schemas to a namespace instance's models collection, they are dropped from the generated swagger spec if they are not used by API endpoints through the use of marshal or expects decorators. This change forces those to be included.

I was running into issues with nested definitions being dropped.

**Comment by j5awry** [comment]
> I was running into issues with nested definitions being dropped.

Just for clarification, were the nested schemas included in schemas that are being used?

brief pseudo:

```
endpoint A uses Schema A
Schema A has nested Schema B
Schema B is not rendered in OpenAPI
```

**Comment by erinpentecost** [comment]
Yes, that's right. I was registering them with schema_model. The parent schema owns a union of child schemas, and the child schemas were not being included.
